set.mm

```diff
@@ -653815,19 +653815,21 @@ D Fn ( I ... ( M - 1 ) ) /\
   ${
     $d w x y z $.
     $( A version of ~ el with an inner existential quantifier on ` x ` , which
-       avoids ~ ax-7 and ~ ax-8 .  (Contributed by SN, 18-Sep-2023.) $)
+       only uses ~ ax-mp to ~ ax-4 , ~ ax-6 , and ~ ax-pr .  (Contributed by
+       SN, 18-Sep-2023.) $)
     sn-el $p |- E. y E. x x e. y $=
-      ( vw vz wel wi wal wex ax-pow weq ax6ev ax9v1 alrimiv eximii exim mpi ) C
-      AECDEFZCGZABEZFAGZSAHZBDBACITRAHUAADJZRAADKUBQCADCLMNRSAOPN $.
+      ( vz weq wo wel wi wal wex ax-pr ax6ev pm2.07 eximii exim mpi ) ACDZPEZAB
+      FZGAHZRAIZBCCBAJSQAITPQAACKPLMQRANOM $.
```

Comment:
Can you keep both versions ?  The old one is not completely subsumed and may be of interest.